src/api/profile/content-types/profile/schema.json

```diff
@@ -34,6 +34,16 @@
     },
     "displayName": {
       "type": "string"
+    },
+    "profilePicture": {
+      "allowedTypes": [
+        "images"
+      ],
+      "type": "media",
+      "multiple": false,
+      "config": {
+        "provider": "azure-storage"
```

Comment:
How would the frontend send the image via the API? Is it a special content type?